etc/lamassu/feedproviders.yml

```
@@ -118,6 +118,12 @@ lamassu:
       codespace: NBE
       url: "http://gbfs.nextbike.net/maps/gbfs/v2/nextbike_eh/gbfs.json"
       language: de
+    - systemId: nextbike_kk
+      operatorId: NBO:Operator:nextbike
```

While we use different code spaces to be sure that vehicle type IDs are different across different feeds, this is not necessary for the operatorId. For Nextbike we use for all feeds `NBK:Operator:nextbike`
```suggestion
      operatorId: NBK:Operator:nextbike
```